client/src/components/CustomMap.js

```diff
@@ -7,18 +8,52 @@ import {
 } from 'react-simple-maps'
 import { scaleLinear } from 'd3-scale'
 
+import {
+  COLOR_LIGHT_GREY,
+  COLOR_MEDIUM_GREY,
+  COLOR_PURPLE,
+  COLOR_PINK,
+} from '../utils/theme'
+
 const wrapperStyles = {
   width: '100%',
   maxWidth: 980,
   margin: '0 auto',
 }
 
-const popScale = scaleLinear()
-  .domain([0, 100000000, 1400000000])
-  .range(['#dbdbdb', '#8c8c8c', '#494949'])
-
 class CustomMap extends Component {
+  constructor() {
+    super()
+    this.handleMouseMove = this.handleMouseMove.bind(this)
+    this.handleMouseLeave = this.handleMouseLeave.bind(this)
```

Comment:
The binding syntax is outdated, we normally just use an arrow function, it'll bind `this` for React. Technically even that technique is outdated in favor of hooks which don't need context anymore.